systems/frontier_user_guide.rst

```diff
@@ -3420,6 +3420,22 @@ The aforementioned behavior is only true for FP atomicAdd() operations. Hardware
 In cases when contention is very low, a FP32 CAS loop implementing an atomicAdd() operation could be faster than an hardware FP32 LDS atomicAdd().
 Applications using single precision FP atomicAdd() are encouraged to experiment with the use of double precision to evaluate the trade-off between high atomicAdd() performance vs. potential lower occupancy due to higher LDS usage.
 
+Library considerations with atomic operations
+---------------------------------------------
+
+Some functionality provided by the rocBLAS and hipBLAS libraries use atomic operations to improve performance by default. This can cause results to not be bit-wise reproducible.
+Functions that may use atomic operations include: gemv, hemv, and symv. These functions introduced atomic operations in ROCm 5.5. Under some circumstances gemm, gemm_ex, trsm, and trsv may also use atomic operations.
```

Comment:
I think this could be a worded a bit clearer, whether they are used is based on problem sizes vs. "under some circumstances".

Reply:
The block recursive gemm based L3 functions call gemm, so they can also use atomic operations. I think it is correct to say all L3 functions may use atomic operations for some problem sizes.

Reply:
The above paragraph could be something like:
Level 2 functions that may use atomic operations include: gemv, hemv, and symv. These functions introduced atomic operations in ROCm 5.5. All the Level 3 functions may use atomic operations. The atomic operations are used for problem sizes where it improves performance.